packages/api/src/controllers/helpers.ts

```diff
@@ -299,12 +299,47 @@ export async function sendgridEmail({
 export async function sendgridEmailPaymentFailed({
   email,
   sendgridApiKey,
-  userId,
+  user,
   invoiceId,
+  invoiceUrl,
+  templateId,
 }) {
+  // If user.lastPaymentFailed is newer than 3 days, don't send email
+  if (
+    user.lastPaymentFailed &&
+    user.lastPaymentFailed > Date.now() - 3 * 24 * 60 * 60 * 1000
```

Comment:
Can you make this a configurable CLI argument or at least a constant in the top of this file?